history/titles/00_k_stormwind.txt

```diff
@@ -495,6 +496,7 @@ c_sentinel_hill = {
 	# 680 abdicated
 	598.3.17={liege=k_stormwind}
 	598.3.17={holder=310560} #Germanicus[patron]
+	603.1.1={holder=5810} #Gryan Stoutmantle
```

Comment:
If you remove patron character from bookmarks completely
either
1) Give gim land elsewhere in Stormwind
or
2) Give county to Gryan in 604.1.1 to have patron landed at least in one bookmark